docs/application/native/guides/multimedia/media-controller.md

```diff
@@ -1200,6 +1202,35 @@ The following table lists all the search category attributes that the server can
 >
 > These attributes support Tizen 5.5 and Higher for Mobile and Wearable.
 
+## Media Controller Result Code Attributes
+
+The following table lists all the result code attributes that the client and server can receive:
+
+**Table: Media controller result code attributes**
+
+| Attribute                               | Description                                             |
+|-----------------------------------------|---------------------------------------------------------|
+| **Result code**                         |                                                         |
+| `MC_RESULT_CODE_SUCCESS`                | The command or the event has been successfully completed. |
+| `MC_RESULT_CODE_ALREADY_DONE`           | The command or the event had already been completed. |
+| `MC_RESULT_CODE_ABORTED`                | The command or the event is aborted by some external event (e.g. aborted play command by incoming call). |
+| `MC_RESULT_CODE_DENIED`                 | The command or the event is denied due to application policy (e.g. cannot rewind in recording). |
+| `MC_RESULT_CODE_NOT_SUPPORTED`          | The command or the event is not supported. |
+| `MC_RESULT_CODE_INVALID`                | The command or the event is out of supported range or the limit is reached. |
+| `MC_RESULT_CODE_TIMEOUT`                | Timeout has occurred.             |
+| `MC_RESULT_CODE_APP_FAILED`             | The application has failed.       |
+| `MC_RESULT_CODE_NO_MEDIA`               | The command or the event has failed because there is no media. |
+| `MC_RESULT_CODE_NO_AUDIO_OUTPUT_DEVICE` | The command or the event has failed because there is no audio output device. |
+| `MC_RESULT_CODE_NO_PEER`                | The command or the event has failed because there is no peer. |
+| `MC_RESULT_CODE_NETWORK_FAILED`         | The network has failed.           |
+| `MC_RESULT_CODE_NO_ACCOUNT`             | The application needs to have an account to which it's logged in. |
+| `MC_RESULT_CODE_LOGIN_FAILED`           | The application could not log in. |
+| `MC_RESULT_CODE_UNKNOWN`                | Unknown error.                    |
+
+> **Note**
```

Comment:
Tizen Custom Note style is updated.

Referencing [the guide](https://github.com/Samsung/tizen-docs/blob/master/styleguide/custom-style.md), please update the notes in this page.

Suggestion. 
There are too many notes in this page.
If it is OK, how about removing the version information notes for old versions?

As this update is targeting Tizen 6.0, for example, keeping previous 2 version information only (5.5, 5.0).
In this case, notes to inform Tizen 4.0 will be removed.
(This feature supports Tizen 4.0 and Higher for Mobile.)

Would you please let me know your opinion?

Reply:
OK, I will update notes in this page.
Removing the version information with keeping 2 version looks good to me.
I will update it together.

Reply:
Thanks for accepting it.

Reply:
thanks too!